console/src/components/Services/Data/TableRelationships/RemoteRelationships/components/EditorCollapsed.tsx

```diff
@@ -19,7 +24,7 @@ const Collapsed: React.FC<Props> = ({ relationship }) => {
         <i>
           {`- ${relationship.table_name} → ${
             relationship.definition.remote_schema
-          } . ${Object.keys(relationship.definition.remote_field)}`}
+          }  ${relationMap()}`}
```

Comment:
Another problem I saw was this: @beerose or @rikinsk either of you could suggest the workaround here.
I think it's a very rare case and only a good to have.

![image](https://user-images.githubusercontent.com/6604943/91582856-0a7f2300-e96e-11ea-9b7f-43e3e0afdeeb.png)